### PR TITLE
`Development`: Add tips about ngx charts to the documentation

### PR DESCRIPTION
**Pull request by willergeorg**
#### General

- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).

### Description

This PR adds some hints together with a small example to the documentation that could be useful when beginning to work with ngx-charts.
Inspect the screenshot below or alternatively have a look at the last GitHub Check.

### Steps for Testing

Look at the changed documentation page (dev/guidelines/client)
#### Code Review
- [x] Review 1
- [x] Review 2

### Test Coverage

nothing changed

### Screenshots

The new section looks like this: 
image

### CODE REVIEW

**Review by MDK5 — DISMISSED** [review]
Found a solution for the highlighting, pygments offers a lexer for html+angular which can be used here.

Changes got addressed, looks good to me now :+1:

**Review by cremertim — APPROVED** [review]
reaprove after the changes

File: docs/dev/guidelines/client.rst [hunk 1]
```diff
@@ -247,4 +251,54 @@ Do not use ``<span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate |
...
+            [results]="ngxData"
+            [scheme]="color"
+            [legend]="false"
+            [xAxis]="true"
+            [yAxis]="true"
+            [yScaleMax]="20"
+            [roundEdges]="true"
+            [showDataLabel]="true">
+            <ng-template #tooltipTemplate let-model="model">
+                {{ labelTitle }}: {{ roundValue((model.value / totalValue) * 100) }}%
```
Comment (Olimbia23) [inline docs/dev/guidelines/client.rst:275]: I think we need to come up with the standardised way of rounding values. I learned recently from @lschlesinger that we have a `round` function defined in utils: https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/shared/util/utils.ts#L61-L80

I think it's worth mentioning.

**Review by Olimbia23 — APPROVED** [review]
Nice work @willergeorg.
Thank you for adding documentation on ngx-charts.
I left one minor comment regarding rounding values in cases of the tooltip.
Approve 👍

File: docs/dev/guidelines/client.rst (hunk 1 above)
Comment (willergeorg) [reply docs/dev/guidelines/client.rst:275]: Thanks for the hint, I will change this immediately.

**Review by Olimbia23 — APPROVED** [review]
Thank you for incorporating the feedback @willergeorg.
Approve 👍

**Review by cremertim — APPROVED** [review]
Documentation seems good to me

File: docs/dev/guidelines/client.rst [hunk 2]
```diff
@@ -76,12 +76,16 @@ Use JSDoc style comments for functions, interfaces, enums, and classes.
...
+
+.. code-block:: html+ng2
+
     <fa-icon [icon]="'times'"></fa-icon>
     <span>Text</span>
 
 Don't do one of these or any other combination of whitespaces:
-.. code-block:: html
+
+.. code-block:: html+ng2
```
Comment (krusche) [inline docs/dev/guidelines/client.rst:87]: why does it change to html+ng2?

File: docs/dev/guidelines/client.rst [hunk 3]
```diff
@@ -247,4 +251,54 @@ Do not use ``<span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate |
...
 Use ``<span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate | artemisTimeAgo }}</span>``
 
+14. Chart Instantiation
+=======================
+
+We are using the framework ngx-charts in order to instantiate charts and diagrams in Artemis.
+
+The following is an example HTML template for a vertical bar chart:
+
+.. code-block:: html+ng2
```
Comment (krusche) [inline docs/dev/guidelines/client.rst:261]: not sure if such a tag for code blocks exists

**Pull request merged**